### DISCUSSION TITLE
opensnitch-ui: command not found although my package manger tells me it's installed

### DISCUSSION THREAD

**Discussion by wh33t** [evilsocket/opensnitch#1028] [Q&A] [answered]
I'm using Linux Mint 21.2

`sudo apt install ./opensnitch*.deb ./python3-opensnitch-ui*.deb`

gives me 

```
$ sudo apt install ./opensnitch*.deb ./python3-opensnitch-ui*.deb
Reading package lists... Done
E: Unsupported file ./python3-opensnitch-ui*.deb given on commandline
```

So I just double click on the downloaded file and it tells me it's installed now.

Then when I try to run I get this.

```
$ opensnitch-ui
opensnitch-ui: command not found
```

Any tips? This seems like a must have feature in a modern Linux OS.

**Comment by gustavo-iniguez-goya** [comment]
Hi @wh33t ,

Sorry for the delay answering this question. 

Where did you downloaded the deb packages from? "unsupported file" smells like a corrupted or invalid deb package.
If you still have it on the disk, could you execute: `~ $ md5sum python*deb` ?

`opensnitch-ui` should be under /usr/bin, you can list all the installed files with : `~ $ dpkg -L python3-opensnitch-ui`

**Comment by wh33t** [comment]
```
$ md5sum opensnitch_1.6.2-1_amd64.deb 
74da4581fd207513d331ed3523f1407a  opensnitch_1.6.2-1_amd64.deb
```

Just re-downloaded from here: https://github.com/evilsocket/opensnitch/releases/download/v1.6.2/opensnitch_1.6.2-1_amd64.deb

```
$ sudo apt install ./opensnitch*.deb ./python3-opensnitch-ui*.deb
Reading package lists... Done
E: Unsupported file ./python3-opensnitch-ui*.deb given on commandline
```

**Reply by gustavo-iniguez-goya** [reply, marked as answer]
mmh, that's the daemon, you need the GUI. Download this deb file from [latest release v1.6.3](https://github.com/evilsocket/opensnitch/releases/tag/v1.6.3): https://github.com/evilsocket/opensnitch/releases/download/v1.6.3/python3-opensnitch-ui_1.6.3-1_all.deb

install it and see if there's any error:
```
~ $ wget https://github.com/evilsocket/opensnitch/releases/download/v1.6.3/python3-opensnitch-ui_1.6.3-1_all.deb
~ $ sudo apt install ./python3-opensnitch-ui_1.6.3-1_all.deb
```

**Comment by wh33t** [comment]
Thanks! Got it working now. 

Sorry, bit of a noob with Github still. Didn't realize I downloaded the wrong thing.

**Comment by wh33t** [comment]
Hey, while I've got ya here. 

Do you know why my outgoing wireguard connection doesn't seem to work with opensnitch enabled? When I disable it the VPN connection works fine. I don't receive any kind of pop up or prompt asking me to allow or deny a connection when I first run wg-quick up ...conf

**Reply by gustavo-iniguez-goya** [reply]
see if you have this hook added:
```
~ # grep xmit /sys/kernel/debug/tracing/kprobe_events 
p:kprobes/piptunnel_xmit iptunnel_xmit
```

If it's not present, what kernel are you using?
On the other hand, you can enable `[x] Debug invalid connections` under Preferences -> Nodes. You should be prompted to allo an "Outgoing connection".
